### PR TITLE
Update the location and usage of ignore filters

### PR DESCRIPTION
**Pull request by Creepow**
https://github.com/retest/recheck/pull/552

### CODE REVIEW

File: docs/recheck/usage/filter.md [hunk 1]
```diff
@@ -40,18 +40,24 @@ By default, filters are used as text files, so that they are reusable within dif
 
 Ignores are a special kind of filters which (by default) are loaded automatically. They define that `true` ignores (i.e. hides) the specified difference, while `false` continues to use it. Furthermore, they have specific names and may be placed in either of the following locations:
 
-1. Globally: `${PROJECT_ROOT}/.retest/`. *Will be created on first run*.
-2. For each Golden Master (i.e. suite). *Must be created manually*.
+1. Globally: `${USER_HOME}/.retest/`. *Must be created manually*.
+2. For each Project: `${PROJECT_ROOT}/.retest/`. *Will be created on first run*.
+3. For each Golden Master indivitually (i.e. suite). *Must be created manually*.
```
Comment (diba1013) [inline docs/recheck/usage/filter.md]:
```suggestion
3. For each Golden Master individually (i.e. suite). *Must be created manually*.
```

File: docs/recheck/usage/filter.md [hunk 2]
```diff
@@ -40,18 +40,24 @@ By default, filters are used as text files, so that they are reusable within dif
...
 ```
-${PROJECT_ROOT}
+${USER_HOME}
 +-- .retest/
-|   +-- recheck.ignore (level 1)
-+-- src/test/resources/retest/recheck/
-    +-- ${SUITE_NAME}
-        +-- recheck.ignore (level 2)
+    +-- recheck.ignore (level 1)
++-- ../${PROJECT_ROOT}
```
Comment (diba1013) [inline docs/recheck/usage/filter.md]: Why `..`?

File: docs/recheck/usage/filter.md [hunk 3]
```diff
@@ -40,18 +40,24 @@ By default, filters are used as text files, so that they are reusable within dif
...
+    +-- .retest/
+    |   +-- recheck.ignore (level 1)
```
Comment (diba1013) [inline docs/recheck/usage/filter.md]: This would be level 2?

File: docs/recheck/usage/filter.md (hunk 2 above)
Comment (Creepow) [reply docs/recheck/usage/filter.md]: Because i would say most people dont have their project on their user home but in some kind of workspace

**Pull request merged**